core-java-modules/core-java-streams-6/src/test/java/com/baeldung/streams/inputstreamintostringstream/InputStreamToStringStreamUnitTest.java

```diff
@@ -0,0 +1,42 @@
+package inputstreamintostringstream;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+import java.util.regex.MatchResult;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+
+public class InputStreamToStringStreamUnitTest {
+
+    byte[] bytes = "Hello\nWorld\nThis\nis\na\ntest".getBytes(StandardCharsets.UTF_8);
+    InputStream inputStream = new ByteArrayInputStream(bytes);
+
+    @Test
+    void givenInputStream_whenConvertingWithBufferedReader_thenConvertInputStreamToStringStream() throws IOException {
+        try (InputStreamReader isr = new InputStreamReader(inputStream, StandardCharsets.UTF_8); BufferedReader reader = new BufferedReader(isr)) {

```

Comment:
The formatting that was here previously was better and already correct:
```suggestion
        try (InputStreamReader isr = new InputStreamReader(inputStream, StandardCharsets.UTF_8);
             BufferedReader reader = new BufferedReader(isr)) {

```